azure-quantum/azure/quantum/target/microsoft/elements/dft/target.py

```diff
@@ -0,0 +1,54 @@
+from azure.quantum.job.base_job import ContentType
+from azure.quantum.job.job import Job
+from azure.quantum.target.target import Target
+from azure.quantum.workspace import Workspace
+from azure.quantum.target.params import InputParams
+from typing import Any, Dict, Type, Union
+from .job import MicrosoftElementsDftJob
+
+
+class MicrosoftElementsDft(Target):
+    """
+    Microsoft Elements Dft target from the microsoft-elements provider.
+    """
+
+    target_names = [
+        "microsoft.dft"
+    ]
+
+
+    def __init__(
+        self,
+        workspace: "Workspace",
+        name: str = "microsoft.dft",
+        **kwargs
+    ):
+        # There is only a single target name for this target
+        assert name == self.target_names[0]
+
+        # make sure to not pass argument twice
+        kwargs.pop("provider_id", None)
+
+        super().__init__(
+            workspace=workspace,
+            name=name,
+            input_data_format="microsoft.xyz.v1",
+            output_data_format="microsoft.dft-results.v1",
+            provider_id="microsoft-elements",
+            content_type=ContentType.json,
```

Comment:
What does this end up doing? Set content-type metadata on the blob? I think text/plain would be more appropriate for the xyz format

Reply:
Good catch, missed this originally. Yes, this sets `Content-Type` metadata on the input data blob. Fixed.